float64 time_scaling_factor
--------------------------


